File: a006.一元二次方程式/main.cpp
#include<iostream>
#include<math.h>
using namespace std;
int main()
{
   int a,b,c,d,e,f;
   while(cin>>a>>b>>c)
   {
       d=sqrt((b*b)-(a*c*4));
       e=(d-b)/(a*2);
       f=(-d-b)/(a*2);

       if(d<0)
       {
           cout<<"No real root ";
       }
       else if(e==f)
       {
           cout<<"Two same roots "<<"x="<<e;
       }
       else if(e!=f)
       {
           cout<<"Two different roots "<<"x1="<<e<<" , x2="<<f;
       }
   }
}

File: a006.一元二次方程式/main.py
a,b,c=map(int,input().split())
d=((b * b) - (a * c * 4))
e = (d**(1/2) - b) / (a * 2)
f = (-(d**(1/2)) - b) / (a * 2)
if d<0:
    print("No real root ")
elif e == f:
    print("Two same roots x="+str(round(e)))
else:
    print("Two different roots "+"x1="+str(round(e))+" , x2="+str(round(f)))